File: ios/VisionCameraColorDetector.m
//
//  VisionCameraColorDetector.m
//  VisionCameraColorDetector
//


#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(VisionCameraColorDetector, NSObject)

RCT_EXTERN_METHOD(noop:(NSString *)message
                 withResolver:(RCTPromiseResolveBlock)resolve
                 withRejecter:(RCTPromiseRejectBlock)reject)

+ (BOOL)requiresMainQueueSetup
{
  return NO;
}

@end

File: ios/VisionCameraColorDetector.swift
//
//  VisionCameraColorDetector.swift
//  VisionCameraColorDetector
//


@objc(VisionCameraColorDetector)
class VisionCameraColorDetector: NSObject {

    @objc(noop:withResolver:withRejecter:)
    func noop(message: String, resolve:@escaping RCTPromiseResolveBlock,reject:@escaping RCTPromiseRejectBlock) -> Void {
        print(message)
    }
}
